src/helpers/color.js

```diff
@@ -24,11 +24,11 @@ export default class Color {
   };
 
   initHsb = () => {
-    const { h, s, l } = this.color.toHsl();
+    const { h, s, v } = this.color.toHsv();
 
     this.hueValue = h;
     this.saturationValue = s;
-    this.lightnessValue = l;
+    this.lightnessValue = v;
```

Comment:
`lightnessValue` it needs to be revised as `brightnessValue` ?

Reply:
Sure, commited changes, also I added brightness as Color object property.